code/modules/paperwork/faxmachine.dm

```diff
@@ -0,0 +1,60 @@
+
+//The IA Agent's Fax Machine
+
+/obj/machinery/faxmachine
+	name = "fax machine"
+	desc = "Used to send messages to your corporate overlords (and promptly get fired for writing fanfiction about their mothers)."
+	circuit = /obj/item/circuitboard/machine/faxmachine
+	icon = 'icons/obj/library.dmi'
+	icon_state = "fax"
+	density = TRUE
+	use_power = IDLE_POWER_USE
+	idle_power_usage = 30
+	active_power_usage = 200
+	power_channel = AREA_USAGE_EQUIP
+	COOLDOWN_DECLARE(static/important_action_cooldown)
+
+/obj/machinery/faxmachine/attack_hand(mob/living/user)
+	. = ..()
+	add_fingerprint(user)
+	if (!COOLDOWN_FINISHED(src, important_action_cooldown))
```

Comment:
This is a personal nitpick but the space between the `if` and the brackets irks me.